recipes/mimalloc/all/patches/0002-include-cstddef-to-get-std-size-t.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/include/mimalloc.h b/include/mimalloc.h
+index 254a7d9..3c3142b 100644
+--- a/include/mimalloc.h
++++ b/include/mimalloc.h
+@@ -377,6 +377,7 @@ mi_decl_nodiscard mi_decl_export void* mi_new_reallocn(void* p, size_t newcount,
+ #ifdef __cplusplus
+ 
+ #include <cstdint>     // PTRDIFF_MAX
++#include <cstddef>     // std::size_t
```

Comment:
May I ask you to create a small PR in the upstream?

Reply:
Yeah! I forgot to create this one.